Detectors/GlobalTracking/src/MatchTOF.cxx

```diff
@@ -569,11 +821,12 @@ void MatchTOF::doMatching(int sec)
     }
     int nStripsCrossedInPropagation = 0; // how many strips were hit during the propagation
     auto& trackWork = mTracksWork[cacheTrk[itrk]];
-    auto& trefTrk = trackWork.getParamOut();
-    auto& intLT = trackWork.getLTIntegralOut();
+    auto& trefTrk = trackWork.first;
+    auto& intLT = mLTinfos[cacheTrk[itrk]];
+
     //    Printf("intLT (before doing anything): length = %f, time (Pion) = %f", intLT.getL(), intLT.getTOF(o2::track::PID::Pion));
-    float minTrkTime = (trackWork.getTimeMUS().getTimeStamp() - mSigmaTimeCut * trackWork.getTimeMUS().getTimeStampError()) * 1.E6; // minimum time in ps
-    float maxTrkTime = (trackWork.getTimeMUS().getTimeStamp() + mSigmaTimeCut * trackWork.getTimeMUS().getTimeStampError()) * 1.E6; // maximum time in ps
+    float minTrkTime = (trackWork.second.getTimeStamp() - mSigmaTimeCut * trackWork.second.getTimeStampError()) * 1.E6;             // minimum time in ps
```

Comment:
actually, the TPC tracks time is not \mus but in TPC time bins, can be converted to ps with multiplicative factor
``o2::tpc::ParameterElectronics::Instance().ZbinWidth * 1e6``

Reply:
I converted it to mus in prepareTPCTracks method to be uniform with what did for global tracks,

Reply:
I will check later if it is convenient to have it in ps for both

Reply:
What I mean is that the tpcTrack.getTime0 and DeltaTBwd, DeltaTFwd are in TPC time bins, not in real time units.

Reply:
I got the point. I converted it to real time in PrepareTPCTracks method wen creating the pair of external track and TimeMUS objects.
Line 576
  o2::dataformats::MCTruthContainer<o2::MCCompLabel> mTPCTrackLabels;                 ///< input TPC tracks MC labels (TPC only tracks)
  o2::dataformats::MCTruthContainer<o2::MCCompLabel>* mTPCTrackLabelsPtr;             ///< input TOF clusters MC labels (pointer to read from tree)

 timeInfo.setTimeStamp(trcOrig.getTime0() * o2::tpc::ParameterElectronics::Instance().ZbinWidth);
timeInfo.setTimeStampError(trcOrig.getDeltaTBwd() * o2::tpc::ParameterElectronics::Instance().ZbinWidth);